pinot-transport/src/main/java/com/linkedin/pinot/routing/RoutingTableSelector.java

```diff
@@ -26,7 +28,7 @@
    * @param realtimeTableName name of the realtime table (e.g. tableName_REALTIME)
    * @return
    */
-  public boolean shouldUseLLCRouting(final String realtimeTableName);
+  boolean shouldUseLLCRouting(final String realtimeTableName);
```

Comment:
i know this existed before this PR, but why should we expose this in interface. Isn't it the responsibility of the routingTableSelector to use LLC or HLC?